LostTreasure/LostTreasure.txt

```diff
@@ -5,15 +5,21 @@
 ; You can read the full terms at https://account.elderscrollsonline.com/add-on-terms
 
 ## Title: |cFFFFBELost Treasure|r
-## Description: The one and only true treasure and survey map locator. All others are just copies.
-## Version: 7.03
-## Author:  |cFFA500CrazyDutchGuy|r
-## APIVersion: 100029
-## SavedVariables: LOST_TREASURE_SV
-## DependsOn: LibAddonMenu-2.0>=30 LibMapPins-1.0>=18 CustomCompassPins
+## Description: The one and only true treasure map and survey report locator. All others are just copies.
+## Version: 8
+## AddOnVersion: 8
+## Author: |cFFA500CrazyDutchGuy|r

```

Comment:
Feel free to add yourself as author to this list. Seeing the amount of work you put in this that would be only fair

Reply:
ok thanks